cmd/puku/puku.go

```diff
@@ -66,27 +66,27 @@ var log = logging.GetLogger()
 var funcs = map[string]func(conf *config.Config, plzConf *please.Config, orignalWD string) int{
 	"fmt": func(conf *config.Config, plzConf *please.Config, orignalWD string) int {
 		paths := work.MustExpandPaths(orignalWD, opts.Fmt.Args.Paths)
-		if err := generate.NewUpdate(true, plzConf).Update(paths...); err != nil {
+		if err := generate.NewUpdate(true, plzConf, paths).Update(paths...); err != nil {
```

Comment:
It just feels wrong to have to stutter like this

Reply:
It feels like we should be able to keep track of the original paths from the Update rather than have to store them in a new updater

Reply:
Agreed and fixed!